packages/block-library/src/query/index.php

```diff
@@ -23,9 +23,19 @@ function render_block_core_query( $attributes, $content, $block ) {
 			// Add the necessary directives.
 			$p->set_attribute( 'data-wp-interactive', true );
 			$p->set_attribute( 'data-wp-navigation-id', 'query-' . $attributes['queryId'] );
+			// Use context to send translated strings.
 			$p->set_attribute(
 				'data-wp-context',
-				wp_json_encode( array( 'core' => array( 'query' => (object) array() ) ) )
+				wp_json_encode(
+					array(
+						'core' => array(
+							'query' => (object) array(
```

Comment:
You don't need object anymore.

```suggestion
							'query' => array(
```